Bottom: b01bfdb2e353a51e67217a448961045f6afc5c30
Top:    30bb7a74e6e2efd1e8fc51f21d3e686b1ce95173
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-14 15:32:50 -0500

Refresh of gitlab-auth

---
diff --git a/docker/buildbot-master/buildbot/master.cfg b/docker/buildbot-master/buildbot/master.cfg
index 06dac1f7..ab6eaae5 100644
--- a/docker/buildbot-master/buildbot/master.cfg
+++ b/docker/buildbot-master/buildbot/master.cfg
@@ -81,10 +81,10 @@ c['www'] = {
         'grid_view': True,
         },
     'auth': util.GitLabAuth(os.getenv('GITLAB_URI'),
-                            os.getenv('GITLAB_CLIENT_ID'),
-                            os.getenv('GITLAB_CLIENT_SECRET'),
+                            os.getenv('GITLAB_OAUTH_ID'),
+                            os.getenv('GITLAB_OAUTH_SECRET'),
                             userInfoProvider=userInfoProvider),
-    'authz':util.Authz(
+    'authz': util.Authz(
         allowRules=[
             util.AnyControlEndpointMatcher(role="admins"),
             ],
